docs/templates/cmake.txt

```diff
@@ -0,0 +1,14 @@
+// Current as of 10/26/2022
+// This defines the role of all the template variables specific to the
+// 'cmake' project type.
+//
+// Please try to keep this alphabetically sorted.
+//
+cmake_minimum_required = Sets the minimum required version for CMake.
+env_dllout = The value of 'dllout' with &#36;() values converted to &#36;ENV{}.
```

Comment:
Is this supposed to have `&#36;`?

Reply:
Yes.  The parser that loads this text does something with dollar signs.

Reply:
I usually read these files directly without processing.  Is it worth a comment or should I just know?

Reply:
I put a comment in the file.